geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1378,6 +1379,18 @@
    */
   String NAME = "name";
   /**
+   * The authentication mode for the protobuf client-server protocol.
+   *
+   * </p>
+   * <U>Description</U>: Specifies the authentication mode used by the geode-protobuf module.
+   * </p>
+   * <U>Default</U>: "NOOP"
+   * </p>
+   * <U>Allowed values</U>: "NOOP" "SIMPLE"
+   */
+  @Experimental
+  String PROTOBUF_PROTOCOL_AUTHENTICATION_MODE = "protobuf-protocol-authentication-mode";
```

Comment:
The property is a String representing the authentication mode to be used by the protobuf protocol. It currently has two valid settings. The protobuf in the name refers to the new client protocol (which we're calling the protobuf protocol).

I don't think we're quite ready to take the feature flag out yet, but we should consider it as this feature nears alpha. Or moving the feature flag to a better config mechanism (DistributionConfig), so that this property and that one are in approximately the same place.